cmake/CompileBoost.cmake

```
@@ -1,12 +1,12 @@
-find_package(Boost 1.67)
+find_package(Boost 1.72)
```

This modification would allow the user to pass in hints via Environmental variable to specify location of Boost.
This change also requires Master to be merged in to define the function **envset**
```suggestion
env_set(Boost_INCLUDE_DIR . PATH "Hint used to locate the Boost version")
find_package(Boost 1.72)
```

---

This is not necessary, instead one can set the environment variable `BOOST_ROOT` to the unpacked source directory